app/javascript/components/upload/UploadExperienceSplitter.jsx

```diff
@@ -18,6 +19,9 @@ function RawUploadExperienceSplitter({
         <div className="splitter-info-outside ">
           <div className='split-title'>
           Choose how to upload expression, conventional metadata, and cluster data
+            { <div className='font-size-14'><a href="https://forms.gle/dRUVSh7WAz9Dh6Ag8" target="_blank" title="Take a brief survey on AnnData data upload">
+              Tell us about your experience with data upload <FontAwesomeIcon icon={faLink}/>
```

Comment:
[Most major products omit icons in external and/or new tab links](https://github.com/broadinstitute/single_cell_portal_core/pull/1716#discussion_r1096296306), and SCP almost always does as well.  Tweaking the copy a bit might also be worthwhile clarity.

```suggestion
              Tell us about your experience with AnnData upload!
```